Makefile

```diff
@@ -14,4 +15,7 @@ $(CENTOS7_TARGETS): .dapper
 $(CENTOS8_TARGETS): .dapper
 	./.dapper -f Dockerfile.centos8.dapper $(@:centos8-%=%)
 
+$(MICROOS_TARGETS): .dapper
+	./.dapper -f Dockerfile.microos.dapper $(@:microos-%=%)
```

Comment:
This Dockerfile is missing, so this target doesn't work yet

```
$ make microos-build
./.dapper -f Dockerfile.microos.dapper build
FATA[0000] stat Dockerfile.microos.dapper: no such file or directory 
make: *** [Makefile:19: microos-build] Error 1
```

Could you also add steps to [.drone.yml](https://github.com/rancher/rancher-selinux/blob/main/.drone.yml) so that this gets automatically built in CI? It looks like RKE has [this](https://github.com/rancher/rke2-selinux/blob/master/.drone.yml#L181-L265)

Reply:
I just added the Dockerfile (just copied the one from rke2-selinux)

Reply:
I'll try to have a look at the drone.yml. Can I just push that to my branch without interfering with any actual releases?

Reply:
Thanks for the review, @cmurphy!

Reply:
Hmm, the parts I added to .drone.yml seem to work, but the last commit is not shown with a green checkmark.

And I get an error regarding "Your license limit is exceeded" when opening the drone page via the "Details" link.